src/msw/joystick.cpp

```diff
@@ -329,16 +329,23 @@ wxString wxJoystick::GetProductName() const
     if (joyGetDevCaps(m_joystick, &joyCaps, sizeof(joyCaps)) != JOYERR_NOERROR)
         return wxEmptyString;
 
+
     wxRegKey key1(wxString::Format(wxT("HKEY_LOCAL_MACHINE\\%s\\%s\\%s"),
                    REGSTR_PATH_JOYCONFIG, joyCaps.szRegKey, REGSTR_KEY_JOYCURR));
 
-    key1.QueryValue(wxString::Format(wxT("Joystick%d%s"),
-                                     m_joystick + 1, REGSTR_VAL_JOYOEMNAME),
-                    str);
+    if (key1.Exists()) {
+        key1.QueryValue(wxString::Format(wxT("Joystick%d%s"),
```

Comment:
Please don't use `wxT()` in the new code, it's unnecessary.

Reply:
The wxT is from the original code. I'm not sure what its purpose is, so will leave it to someone else to remove it!